openapi/components/schemas/Storefront/StorefrontPlan.yaml

```
@@ -1,12 +1,107 @@
-allOf:
-  - $ref: ../Common/CommonPlan.yaml
-  - properties:
-      _embedded:
+type: object
+required:
+  - name
+  - currency
+  - productId
+  - pricing
+properties:
+  id:
+    type: string
+    description: ID of the plan.
+    maxLength: 50
+    example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
+  name:
+    description: |-
+      Name of the plan.
+      This name is displayed on invoices and receipts.
+    type: string
+    maxLength: 255
+  description:
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
+      such as: bold, underline, italic, and hyperlinks.
```

A nitpick, I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```